src/main/java/com/coolkids/coolKidsApp/controllers/EventController.java

```diff
@@ -17,7 +17,6 @@
 @RequestMapping(path = "api/v1/")
 @AllArgsConstructor
 public class EventController {
-
     private final EventService eventService;
 
     @PostMapping("/events") //Getmapping vs Postmapping here?
```

Comment:
If you are trying to get the events, then it would be a get request. So @GetMapping annotation

Reply:
Thanks, I changed it.